src/cclib/parser/adfparser.py

```diff
@@ -136,10 +139,20 @@ def extract(self, inputfile, line):
             while line[:5] != "title":
                 line = inputfile.next()
 
+        if "Amsterdam Density Functional" in line:
+            self.metadata['package_version'] = line.split()[5]
+
         if line[1:10] == "Symmetry:":
             info = line.split()
             if info[1] == "NOSYM":
                 self.nosymflag = True
+                point_group_full = "c1"
+            else:
+                point_group_full = info[1].replace('(', '').replace(')', '').lower()
+            # TODO
+            point_group_abelian = point_group_full
+            self.metadata['symmetry_full'] = point_group_full
+            self.metadata['symmetry_abelian'] = point_group_abelian

```

Comment:
What is the point of having symmetry_abelian if it's never different here?

Also, shouldn't it be a Boolean, since a group is simply Abelian or not? And since it is a property of the group itself, do we really need to parse it?

Reply:
I think `symmetry_used` is was the best. What do you think now?